dex-operator-image/dex-operator-image.kiwi.ini

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- OBS-AddTag: _PRODUCT_/dex-operator:%%LONG_VERSION%% _PRODUCT_/dex-operator:%%LONG_VERSION%%-<RELEASE> -->
+
+<image schemaversion="6.5" name="_PRODUCT_-dex-operator">
+  <description type="system">
+    <author>SUSE Containers Team</author>
+    <contact>containers@suse.com</contact>
+    <specification>Dex operator running on a _DISTRO_ container guest</specification>
+  </description>
+  <preferences>
+    <type
+      image="docker"
+      derived_from="obsrepositories:/_BASEIMAGE_">
+      <containerconfig
+        name="_PRODUCT_/dex-operator"
+        tag="%%SHORT_VERSION%%"
+        maintainer="SUSE Containers Team &lt;containers@suse.com&gt;">
+        <entrypoint execute="/usr/local/bin/dex-operator">
```

Comment:
I think this can be rewritten as

```<entrypoint execute="/usr/local/bin/dex-operator"/>```

and drop the `</entrypoint/>` below

Reply:
Done